[TF 2.0 API Docs] tf.data Add pointer to tutorials which work with r2.0

netskink opened the pull request.

goldiegadde committed:
Update the version of TF to 2.0.0-alpha0

gunan committed:
Merge pull request #26326 from tensorflow/2.0-ff

2.0 ff , Move r2.0 branch ahaed to pick up test and build fixes.

yifeif committed:
Install auditwheel 1.5.0.

PiperOrigin-RevId: 236681136

gunan committed:
Merge pull request #26272 from tensorflow/version-update-ggadde

Update the version of TF to 2.0.0-alpha0

martinwicke committed:
Use eager-friendly evaluation for bincount_op_test.

PiperOrigin-RevId: 236690668

tensorflower-gardener committed:
Internal change

PiperOrigin-RevId: 236729117

tensorflower-gardener committed:
Add ops_history.v2.pbtxt for TF 2.0 alpha release.

PiperOrigin-RevId: 236732665

mihaimaruseac committed:
Disable some v2 py36 pip flaky tests

PiperOrigin-RevId: 236747310

tensorflower-gardener committed:
Internal Change

PiperOrigin-RevId: 236754902

tensorflower-gardener committed:
disable failing virtual_gpu_test on gpu.

PiperOrigin-RevId: 236780392

tensorflower-gardener committed:
Add ops_history.v2.pbtxt to the build dependancy of
backwards_compatibility_test.

PiperOrigin-RevId: 236782576

tensorflower-gardener committed:
Work around crash in nvcc by using the same #include files and order as max_pooling, which seems to not trigger the crash.

PiperOrigin-RevId: 236832623

gunan committed:
Merge pull request #26367 from goldiegadde/cherrypicks_AEHRG

Cherrypick build and test fixes for fixing the release builds for TF 2.0 alpha-0

yifeif committed:
Switch to use pip3

gunan committed:
Merge pull request #26370 from tensorflow/yifeif-patch-2

Switch to use pip3

tensorflower-gardener committed:
Disable bincount_op_test on windows gpu.

PiperOrigin-RevId: 236907827

goldiegadde committed:
Update TF 2.0.0-alpha-0 release notes. (#26369)

Release Notes for 2.0.0-alpha0

yifeif committed:
Merge pull request #26373 from goldiegadde/cherrypicks_MQMPF

Disable bincount_op_test on windows gpu.

angerson committed:
Pull support for latest Bazel version

This makes the 2.0 branch match the current maximum Bazel version settings on `master`. It should help resolve https://github.com/tensorflow/tensorflow/issues/26553, where the latest `devel` images don't support `r2.0`.

yifeif committed:
Merge pull request #27399 from tensorflow/angersson-r2.0-bazel-version

Pull support for latest Bazel version

angerson committed:
Fix configure.py to properly compare "X.Y.Z" with "X.Y"

Right now, "0.24" is treated as lower than "*.*.*" because of the odd comparison method that adds digits to each existing section. This change converts "0.24" to "0.24.0" to fix that.

This will probably need to be pulled into r2.0.

PiperOrigin-RevId: 241950768

yifeif committed:
Merge pull request #27515 from tensorflow/cherrypick-87ea41d023a985c5ecee0370c7f7381c8a8cee52

Fix configure.py to properly compare "X.Y.Z" with "X.Y"

netskink committed:
Merge pull request #1 from tensorflow/master

pull from official

netskink committed:
Adding pointers to the tutorials

netskink committed:
Added dataset example

Added an example using canned data which can be fed into a keras.model_fit() call

netskink committed:
Added the model.fit() line to example

So that the example met the normal use case

alextp commented on tensorflow/python/data/ops/dataset_ops.py:
````diff
@@ -92,6 +92,71 @@ class DatasetV2(tracking_base.Trackable, composite_tensor.CompositeTensor):
+For example:
+
+    ```python
+    tf.print("tf.keras.model.fit() can take a single tf.data.Dataset for training features and labels. \
````
Should also mention that if you use tf.keras.layers.Input(..., name=name) the dataset can also return a dictionary with the names of the inputs as keys.

netskink replied:
I was unaware of that capability!  But I’m excited to try it out.  I appreciate the response and I’ll edit and resubmit.

netskink replied:
I was unaware of that capability!  But I’m excited to try it out.  I appreciate the response and I’ll edit and resubmit.

alextp replied:
Please ping here when you want me to take another look :-)

goldiegadde committed:
Merge branch 'r2.0' of github.com:tensorflow/tensorflow into r2.0

gunan committed:
Merge pull request #29370 from goldiegadde/r2.0

R2.0 fastforward branch.

tensorflower-gardener committed:
Disable tests that are failing in TFv2 Windows CPU/GPU builds.

PiperOrigin-RevId: 251343281

k-w-w committed:
Keras models and layers saving and reviving code. Implements go/tf-model-serialization.

To save and revive a model:
1. Save the model using tf.saved_model.save
2. call load_from_save_model_v2

This restores various metadata about Keras models and layers, as well as their call and loss functions.

Changes to object serialization:
- Adds private fields for tracking object's identifier and metadata.
- Added _list_extra_dependencies_for_serialization, which allows objects to save extra
  dependencies when serialized to SavedModel.
- Object graph view maintains a serialization cache object that is passed to each object when serializing functions/extra dependencies.

PiperOrigin-RevId: 251386039

jaingaurav committed:
Add get_ prefix to parallelism_threads APIs

PiperOrigin-RevId: 251391606

goldiegadde committed:
Version update.

tensorflower-gardener committed:
Disable tests that are failing in TFv2 Windows CPU/GPU builds.

PiperOrigin-RevId: 251441123

goldiegadde committed:
fix the merge conflicts.

goldiegadde committed:
the cherrypick of this file had some issues as some changes were rolled
back. Cleaning up this file, so that only kathy's changes are in.

yifeif committed:
Merge pull request #29415 from goldiegadde/ggadde-2

Cleaning up this file, so that only kathy's changes are in.

netskink committed:
Merge branch 'r2.0' of https://github.com/tensorflow/tensorflow

lamberta commented:
Links added automatically now. Thanks
